### PR TITLE
Initial Implementation

### PR DESCRIPTION
**Pull request by cuevaskoch**
* RestEase client for WaySine JSON endpoints
* Standardized docs from DX template
* Cake build scripts, semver and nuget.

### CODE REVIEW

File: src/Syncromatics.Clients.WaySine/PolledDateTimeConverter.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+{
+    public class PolledDateTimeConverter : DateTimeConverterBase
+    {
+        public override bool CanWrite => false;
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            if (reader.TokenType != JsonToken.Integer) return null;
+            var ticks = (long)reader.Value;
+            return new DateTime(1970, 1, 1).AddMilliseconds(ticks);
```
Comment (derrickcrowne) [inline src/Syncromatics.Clients.WaySine/PolledDateTimeConverter.cs]: I recommend using DateTimeOffset.FromUnixTimeMilliseconds()

Comment (cuevaskoch) [reply src/Syncromatics.Clients.WaySine/PolledDateTimeConverter.cs]: That's much better.  I'll add it.  Thanks!

**Review by thzinc — APPROVED** [review]

File: build.cake [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    {
+        CleanDirectory(buildDir);
+        CleanDirectory(testDir);
+    });
+
+Task("Build")
+    .IsDependentOn("InnerTest");
+
+Task("Package")
+    .IsDependentOn("InnerPackage");
```
Comment (kennethito) [inline build.cake:65]: What's the reason for this again?

**Review by kennethito — APPROVED** [review]

File: build.cake (hunk 2 above)
Comment (cuevaskoch) [reply build.cake:65]: I copied the tasks from an existing repository.  I think it was initially spinning up its own docker container and doing the Inner* tasks in there, but that's no longer the case.  @thzinc do we still need the Inner* tasks for anything?

**Pull request merged**